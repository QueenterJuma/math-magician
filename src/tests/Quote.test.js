import React from 'react';
import renderer from 'react-test-renderer';
import FetchQuote from '../component/Quote.js';

describe('FetchQuote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FetchQuote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
