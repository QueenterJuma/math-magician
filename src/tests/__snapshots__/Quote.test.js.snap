// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchQuote renders correctly 1`] = `
<div
  className="output"
>
  <h3
    className="quote-heading"
  >
    Quote of the day
  </h3>
  <div />
</div>
`;
